### PR TITLE
Add more instructions on how to delete orgs 

### PR DESCRIPTION
**Pull request by miveronese**
Sourcegraph employees that have created organization on dotcom for testing and learning purposes needs to know how do delete their organizations. This PR adds instructions for that. 

For a better view of the changes:[ click here](https://github.com/sourcegraph/handbook/pull/4028/files?short_path=f186949#diff-f1869495a84f64b7209ba4177aec5a4e0d1f79ae1705bee431be4fb36b608cc1)

### CODE REVIEW

**Comment by miveronese** [comment]
Closing this for a few minutes because I realize there's one quick update needed

File: content/departments/engineering/dev/process/cloud_processes.md [hunk 1]
```diff
@@ -41,3 +41,38 @@ DELETE * FROM orgs WHERE id = $ORG_ID;
...
+
+## Closing down an employee's organization on Sourcegraph Cloud
+
+If you have created an organization on Sourcegraph Cloud (in production) for yourself that you’ve been using for testing or learning, follow these instructions when it’s time to delete it:
+
+- Go to your organization settings page. You can access it from the top right menu where there's a link to your org.
+
+- You should see the "Delete this organization" button in the bottom of the page. Go ahead and click it, confirming the deletion by typing the org's name in the input field.
+
+  Note that this action can't be undone and will hard delete the organization along with all resources associated to it, such as:
```
Comment (unknwon) [inline content/departments/engineering/dev/process/cloud_processes.md]:
```suggestion
  Note that this action can't be undone and will hard delete the organization along with all resources associated to it, including:
```

Non-native speaker, but I think "such as" is often used to make examples, and make me feel that we aren't sure about what will be deleted ourselves.

File: content/departments/engineering/dev/process/cloud_processes.md [hunk 2]
```diff
@@ -41,3 +41,38 @@ DELETE * FROM orgs WHERE id = $ORG_ID;
...
+
+  - Organization members
```
Comment (unknwon) [inline content/departments/engineering/dev/process/cloud_processes.md]:
```suggestion
  - Organization membership
```

Unless we are deleting members' accounts as well.

File: content/departments/engineering/dev/process/cloud_processes.md [hunk 3]
```diff
@@ -41,3 +41,38 @@ DELETE * FROM orgs WHERE id = $ORG_ID;
 ### Other useful queries
 
 A list of useful DB queries can be found in [useful DB queries](https://docs.google.com/spreadsheets/d/1Z1-7uJwtF2etZFeqTcJS4z9WcAjKlkCxt8HQ931D3dA).
+
+## Closing down an employee's organization on Sourcegraph Cloud
+
+If you have created an organization on Sourcegraph Cloud (in production) for yourself that you’ve been using for testing or learning, follow these instructions when it’s time to delete it:
+
+- Go to your organization settings page. You can access it from the top right menu where there's a link to your org.
```
Comment (unknwon) [inline content/departments/engineering/dev/process/cloud_processes.md]: nit: I think we should be using consistent wording at least within the same document.

```suggestion
- Go to your organization settings page. You can access it from the top right menu where there's a link to your organization.
```

File: content/departments/engineering/dev/process/cloud_processes.md [hunk 4]
```diff
@@ -41,3 +41,38 @@ DELETE * FROM orgs WHERE id = $ORG_ID;
...
+  - Organization invitations
+  - Registry extensions
+  - Saved searches
+  - Notebooks
+  - Settings
+  - Code insights
+  - Batch changes
+  - Code monitors
+
+Deleting an organization will also remove all its synced repositories from Sourcegraph.
```
Comment (unknwon) [inline content/departments/engineering/dev/process/cloud_processes.md]:
```suggestion
Deleting an organization will also remove all its synced repositories from Sourcegraph Cloud.
```

**Review by unknwon — APPROVED** [review]
Approve to unblock, left few comments.

File: content/departments/engineering/dev/process/cloud_processes.md (hunk 1 above)
Comment (miveronese) [reply content/departments/engineering/dev/process/cloud_processes.md]: Non-native speaker too. Thanks for the suggestion, @unknwon :)

**Pull request merged**